src/mpyl/steps/deploy/bpm/modeler.py

```diff
@@ -0,0 +1,59 @@
+"""Camunda modeler related methods to deploy diagrams"""
+
+import os
+from logging import Logger
+from collections import namedtuple
+from .camunda_modeler_client import CamundaModelerClient
+
+File = namedtuple("File", ["name", "file_id", "revision"])
+
+
+def deploy_diagram_to_modeler(
+    logger: Logger, bpm_file_path: str, project_id: str, client: CamundaModelerClient
+) -> None:
+    for file_name in (
+        [fn for fn in os.listdir(bpm_file_path) if fn.endswith(".bpmn")]
+        if os.path.isdir(bpm_file_path)
+        else []
+    ):
+        logger.info(file_name)
```

Comment:
I would log something more descriptive here than just the filename, like "Updating diagram: {file_name}"

Reply:
This still seems to be the same?

Reply:
my bad, I didnot save it